README.md

```diff
@@ -36,3 +36,17 @@ See the Eclipse Che Documentation about [how you can join our community](https:/
 ## Builds
 
 * [![release latest stable](https://github.com/eclipse-che/che-server/actions/workflows/release.yml/badge.svg)](https://github.com/eclipse-che/che-server/actions/workflows/release.yml)
+
+### Develocity
+
+[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://develocity-staging.eclipse.org/)
+
+The `che-server` builds are revved up by [Develocity](https://develocity-staging.eclipse.org/scans), which enhances developer productivity through features like Build Scan for build insights and collaborative troubleshooting and remote Build Cache for build acceleration. It further provides comprehensive reporting and visualization of build data.
+
+You can find the Build Scan produced by a build by clicking on the generated link at the end of the build, both locally and on CI (given that you are authenticated to publish). You can find the all scans published by the `che-server` builds on the [Eclipse Develocity dashboard](https://develocity-staging.eclipse.org) by [searching](https://develocity-staging.eclipse.org/scans?search.rootProjectNames=che%20server) for the `che server` project.
```

Comment:
CI is client interface? 

What can I publish if I'm authenticated? The resulting Build Scan?

Reply:
CI is the Continuous Integration server, which in the case of `che-server` is Github Actions. For clarity, that can also be renamed to GitHub Actions in the README doc.

When authenticated, you can publish scans from local builds. Without authentication, you can read from the remote build cache (which is written to by the builds on Github Actions). If you want to configure other features ([Test Distribution](https://docs.gradle.com/develocity/test-distribution/), or [Predictive Test Selection](https://docs.gradle.com/develocity/predictive-test-selection/)), which are also provided by Develocity and enabled for the Eclipse Foundations projects, you would also need to be authenticated to use them locally.

Reply:
@deerskindoll @ribafish  `CI` -> `Github Action CI`?

Reply:
Sounds good